### PR TITLE
FillMaskPipeline: support passing top_k on __call__

### PR DESCRIPTION
**Pull request by julien-c**
Also change name from topk to top_k for more consistency with the TextGenerationPipeline

### CODE REVIEW

**Review by patrickvonplaten — APPROVED** [review]

**Comment by julien-c** [comment]
Ok so should be ready to merge after a quick review @LysandreJik @sgugger!

File: src/transformers/pipelines.py [hunk 1]
```diff
@@ -1195,7 +1196,8 @@ def __init__(
         framework: Optional[str] = None,
         args_parser: ArgumentHandler = None,
         device: int = -1,
-        topk=5,
+        top_k=5,
+        topk: Optional[int] = None,
```
Comment (sgugger) [inline src/transformers/pipelines.py:1200]: Usually we put the deprecated arguments in the kwargs so they are not visible anymore.

**Review by sgugger — APPROVED** [review]
LGTM!
Just one nit, we usually hie the deprecated arguments in the kwargs (and so you have to pop them after).

File: src/transformers/pipelines.py (hunk 1 above)
Comment (julien-c) [reply src/transformers/pipelines.py:1200]: Ok will do

**Pull request merged**